inference/core/workflows/execution_engine/executor/output_constructor.py

```diff
@@ -15,6 +19,9 @@ def construct_workflow_output(
     workflow_outputs: List[JsonField],
     execution_cache: ExecutionCache,
 ) -> Dict[str, List[Any]]:
+    # TODO: figure out if we needed generic "to-root coordinates" transformation?
+    # Maybe output constructor should never touch this? Maybe it's better to
+    # have steps transforming to specific coordinates systems?
```

Comment:
should we extract this to separate task? It seems we are already handling this by calling `sv_detections_to_root_coordinates` below

Reply:
issue submitted